service/src/test/java/org/databiosphere/workspacedataservice/service/model/RecordTypeSchemaTest.java

```diff
@@ -37,4 +40,17 @@ void testContainsAttribute() {
     assertTrue(schema.containsAttribute("attr1"));
     assertFalse(schema.containsAttribute("doesNotExist"));
   }
+
+  @Test
+  void testGetAttributeSchema() {
+    assertEquals(schema.getAttributeSchema("attr1"), new AttributeSchema("attr1", "STRING", null));
```

Comment:
```suggestion
    assertEquals(schema.getAttributeSchema("attr1"), new AttributeSchema(/* name= */ "attr1", /* datatype= "STRING", /* relatesTo= */ null));
```

I know the convention throughout the codebase is to just instantiate `AttributeSchema` as you have here, but where possible, and in the absence of a more expressive construction process like a builder, I recommend putting in this kind of arg comment to make it more scannable when the arg value itself isn't intention revealing enough to distinguish what the arg is.

Reply:
For what it's worth, Intellij has been helpfully showing me the argument names.

![Screenshot 2024-01-19 at 12 45 18 PM](https://github.com/DataBiosphere/terra-workspace-data-service/assets/1156625/67a6ed33-0c31-417e-8d45-30be0ac65c2f)

Reply:
Done in https://github.com/DataBiosphere/terra-workspace-data-service/pull/463/commits/495b3ebc1c5fb6c7c32205d27be491c41a586296.